src/Microsoft.VisualStudio.Editors/PropPages/ReferencePropPage.vb

```diff
@@ -792,10 +793,11 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 Dim solution = visualStudioWorkspace.CurrentSolution
                 Dim CurrentImportNames As New List(Of String)
 
-                For Each projectId In solution.ProjectIds
-                    ' We need to find the project that matches by IVsHierarchy
-                    If visualStudioWorkspace.GetHierarchy(projectId) Is ProjectHierarchy Then
-                        Dim compilationTask = solution.GetProject(projectId).GetCompilationAsync(cancellationTokenSource.Token)
+                For Each project In solution.Projects
```

Comment:
To get this working correct, we need to do some work to get to the actual IVSHierarchy, created by CPS, from ConfiguredProjectHostObject IVSHierarchy. Seems this will be used by other components as well

fyi, @tmeschter @srivatsn